adapter_pipelines/Optimus/adapter.wdl

```diff
@@ -36,6 +36,9 @@ task GetInputs {
   }
   output {
     String sample_id = read_string("sample_id.txt")
+    File tar_star_reference = read_string("tar_star_reference.txt") # star reference
+    File annotations_gtf = read_string("annotations_gtf.txt") # gtf containing annotations for gene tagging
+    File ref_genome_fasta = read_string("ref_genome_fasta.txt") # genome fasta file
```

Comment:
I'm not sure how are these vars passed into the `call Optimus.Optimus as analysis`, should we use something like `tar_star_reference = prep.tar_star_reference` there?

Reply:
Same question for cellranger